toqito/state_metrics/fidelity_of_separability.py

```diff
@@ -0,0 +1,194 @@
+"""Add function for fidelity of separability as defined in [Phil23]_.
+
+The constrainsts for this function are positive partial transpose(PPT)
+& k-extendible states.
+"""
+
+from toqito.state_props import is_pure, has_symmetric_extension
+from toqito.matrix_props import is_density
+import picos
+import numpy as np
+from toqito.perms import symmetric_projection
```

Comment:
Nitpick: group the `toqito` imports together.

Another note for future me is to leverage the `isort` package to do this automatically.